conans/client/generators/cmake_common.py

```diff
@@ -181,6 +183,28 @@ def generate_targets_section(dependencies):
     set(${libraries_abs_path} ${CONAN_FULLPATH_LIBS} PARENT_SCOPE)
 endfunction()
 
+function(conan_package_library_targets libraries package_libdir libraries_abs_path deps build_type)
+    foreach(_LIBRARY_NAME ${libraries})
+        unset(CONAN_FOUND_LIBRARY CACHE)
+        find_library(CONAN_FOUND_LIBRARY NAME ${_LIBRARY_NAME} PATHS ${package_libdir}
+                     NO_DEFAULT_PATH NO_CMAKE_FIND_ROOT_PATH)
+        if(CONAN_FOUND_LIBRARY)
+            message(STATUS "Library ${_LIBRARY_NAME} found ${CONAN_FOUND_LIBRARY}")
+            set(_LIB_NAME ${_LIBRARY_NAME}${build_type})
+            add_library(${_LIB_NAME} UNKNOWN IMPORTED)
```

Comment:
Prefix lib name with the package name? Maybe same lib name from different packages, it could work with abspaths but will fail if the target name is the same.